networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,44 @@ The Network Observability Operator is now stable and the release channel is upgr
+
+[id="network-observability-operator-1.2.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, after changing the `namespace` value in the FlowCollector spec, eBPF Agent pods running in the previous namespace were not appropriately deleted. Now, the pods running in the previous namespace are appropriately deleted. (link:https://issues.redhat.com/browse/NETOBSERV-774[NETOBSERV-774])
+
+* Previously, after changing the `caCert.name` value in the FlowCollector spec (such as in Loki section), FlowLogs-Pipeline pods and Console plug-in pods were not restarted, therefore they were unaware of the configuration change. Now, the pods are restarted, so they get the configuration change. (link:https://issues.redhat.com/browse/NETOBSERV-772[*NETOBSERV-772*])
+
+* Previously, network flows between pods running on different nodes were sometimes not correctly identified as being duplicates because they are captured by different network interfaces. This resulted in over-estimated metrics displayed in the console plug-in. Now, flows are correctly identified as duplicates, and the console plug-in displays accurate metrics. (link:https://issues.redhat.com/browse/NETOBSERV-755[*NETOBSERV-755*])
+
+* The "reporter" option in the console plug-in is used to filter flows based on the observation point of either source node or destination node. Previously, this option mixed the flows regardless of the node observation point. This was due to network flows being incorrectly reported as Ingress or Egress at the node level.  Now, the network flow direction reporting is correct. The "reporter" option filters for source observation point, or destination observation point, as expected. (link:https://issues.redhat.com/browse/NETOBSERV-696[*NETOBSERV-696*])
+
+* Previously, for agents configured to send flows directly to the processor as GRPC+protobuf requests, the submitted payload could be too large and is rejected by the processors' GRPC server. This occured under  very-high-load scenarios and with only some configurations of the agent. The agent logged an error message, such as: _grpc: received message larger than max_. As a consequence, there was information loss about those flows.
+Now, the GRPC payload is split into several messages when the size exceeds a threshold. As a result, the server maintains connectivity. (link:https://issues.redhat.com/browse/NETOBSERV-617[*NETOBSERV-617*])
+
+[id="network-observability-operator-1.2.0-known-issues"]
+=== Known issue
+* In the 1.2.0 release of the Network Observability Operator, a Loki CA certificate issue periodically affects `flowlogs-pipeline` pods and results in dropped flows rather than flows written to Loki. The problem seems to self-correct, but it still causes flow data loss. (link:https://issues.redhat.com/browse/NETOBSERV-980[*NETOBSERV-980*])
```

Comment:
@skrthomas what about:
```suggestion
* In the 1.2.0 release of the Network Observability Operator, using Loki Operator 5.6, a certificate issue periodically affects `flowlogs-pipeline` pods and results in dropped flows rather than flows written to Loki. The problem self-correct after some time but causes flow logs loss during the transition. (link:https://issues.redhat.com/browse/NETOBSERV-980[*NETOBSERV-980*])
```

Reply:
okay, those additional details are perfect. Will add them. TY!

Reply:
What is the "transition" happening? Is it certificate transitions?

Reply:
@skrthomas the transition is happening for FLP to load newly rotated certificates.